In-house patch to remove unnecessary dependencies from Heat's
requirements files. The specific reasons are as follows:

argparse	No longer applicable

httplib2	No longer applicable

pycrypto	Not applicable to Solaris (M2Crypto used instead)

--- heat-2013.2.3/heat.egg-info/requires.txt.orig	2014-04-03 11:47:55.000000000 -0700
+++ heat-2013.2.3/heat.egg-info/requires.txt	2014-07-07 03:03:36.128102921 -0700
@@ -1,8 +1,6 @@
 pbr>=0.5.21,<1.0
-pycrypto>=2.6
 eventlet>=0.13.0
 greenlet>=0.3.2
-httplib2
 iso8601>=0.1.8
 kombu>=2.4.8
 lxml>=2.3
@@ -22,4 +20,4 @@
 PyYAML>=3.1.0
 paramiko>=1.8.0
 Babel>=1.3
-oslo.config>=1.2.0
\ No newline at end of file
+oslo.config>=1.2.0
--- heat-2013.2.3/requirements.txt.orig	2014-04-03 11:44:49.000000000 -0700
+++ heat-2013.2.3/requirements.txt	2014-07-07 03:03:32.472531751 -0700
@@ -1,11 +1,8 @@
 pbr>=0.5.21,<1.0
-pycrypto>=2.6
 eventlet>=0.13.0
 greenlet>=0.3.2
-httplib2
 iso8601>=0.1.8
 kombu>=2.4.8
-argparse
 lxml>=2.3
 netaddr
 six>=1.4.1
